### PR TITLE
Kz/rf/pom.Verify "My Views" link in the user dropdown menu is clickable

### PR DESCRIPTION
**Pull request by Kseniyaza**
https://trello.com/c/Cf6p09Lu/1858-rf0106004-header-verify-my-views-link-in-the-user-dropdown-menu-is-clickable

### CODE REVIEW

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import searchBoxDocumentationPageData from "../../fixtures/pom_fixtures/searchBo
 import userBuildsPageData from "../../fixtures/pom_fixtures/userBuildsPage.json";
 import UserProfilePageData from "../../fixtures/pom_fixtures/userProfilePage.json";
 import restApiDocPageData from "..//..//fixtures/pom_fixtures/restApiDocPage.json";
+import { realHover } from "cypress-real-events/commands/realHover";
```
Comment (Nzs1) [inline cypress/e2e/tests/headerAndFooter.cy.js:16]: I think you have to import as mentioned in https://redroverschool.slack.com/archives/C049HHDG0KF/p1686201331584039

File: cypress/e2e/addDescriptionToThePipeline.cy.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ describe.skip('addDescriptionToThePipeline', () =>{
         cy.get('.jenkins-breadcrumbs__list-item').click();
         cy.get('.jenkins-table__link > span').should('include.text', `${pipelineName.namePipeline}`).should('exist');
     })
-    it('addDescriptionToThePipeline', () =>{
+    it.only('addDescriptionToThePipeline', () =>{
```
Comment (Nzs1) [inline cypress/e2e/addDescriptionToThePipeline.cy.js:16]: I think you forgot to remove 'only'

**Review by SawkaQA — APPROVED** [review]

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 3]
```diff
@@ -214,4 +215,12 @@ describe('headerAndFooter', () => {
                 expect($el.text()).to.contain(restApiDocPageData.RestApiDocPageItemsList[idx])
             })
     })
+
+    it('TC 17.04 | Open "My views" tab from user dropdown', () => {
+        headerAndFooter
+            .clickDropDownMenuButton(realHover)
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:221]: And what is the `realHover` argument

File: cypress/pageObjects/HeaderAndFooter.js [hunk 4]
```diff
@@ -136,6 +139,15 @@ class HeaderAndFooter {
         return new HomePage();
     }
 
-}
+    clickDropDownMenuButton() {
+        this.getDropDownMenuButton().realHover().click();
+        return this
+    }
```
Comment (Rom-T) [inline cypress/pageObjects/HeaderAndFooter.js:145]: there is already such a method above `clickUserDropDownBtn()`

File: cypress/pageObjects/HeaderAndFooter.js [hunk 5]
```diff
@@ -136,6 +139,15 @@ class HeaderAndFooter {
...
+
+    clickDropDownnMenuMyViews() {
+        this.getDropDownnMenuMyViews().click();
+        return new MyViewPage();
+    }
```
Comment (Rom-T) [inline cypress/pageObjects/HeaderAndFooter.js:150]: there is already such a method above `selectUserMyViewsMenu()`

File: cypress/e2e/tests/headerAndFooter.cy.js (hunk 1 above)
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:16]: delete please this import

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 6]
```diff
@@ -214,4 +215,12 @@ describe('headerAndFooter', () => {
...
+            .clickDropDownnMenuMyViews()
+        cy.url().should('contain', 'view/all/')
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:223]: please move  cy.url() in the page MyViewPage

**Review by SawkaQA — APPROVED** [review]

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 7]
```diff
@@ -214,4 +214,11 @@ describe('headerAndFooter', () => {
                 expect($el.text()).to.contain(restApiDocPageData.RestApiDocPageItemsList[idx])
             })
     })
+
+    it('TC 17.04 | Open "My views" tab from user dropdown', () => {
+        headerAndFooter
+            .clickUserDropDownBtn()
+            .selectUserMyViewsMenu()
+             cy.url().should('contain', 'view/all/')
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:222]: Don't break the chain. You need to create a url on the page, as well as transfer the data to fixtures.

**Comment by Rom-T** [comment]
Either correct all comments or close the PR. As the project continues to evolve, your test will soon become obsolete.

**Pull request closed without merging**